Activating Open Collective

xdamman opened the pull request:
Please merge this pull request to activate your Open Collective https://opencollective.com/bower

sheerun commented on package.json:
```diff
@@ -91,10 +92,15 @@
     "coveralls": "coveralls",
-    "prepublish": "in-publish && echo 'You need to use \"grunt publish\" to publish bower' && false || not-in-publish"
+    "prepublish": "in-publish && echo 'You need to use \"grunt publish\" to publish bower' && false || not-in-publish",
+    "postinstall": "./node_modules/.bin/opencollective-postinstall || exit 0"
```
Does `exit 0` work on windows?

sheerun commented:
I cannot merge it, it breaks installs on node 0.10-0.12

xdamman commented:
🤔 good question. Do you have a Windows machine to test this?

sheerun commented:
@xdamman We use Appveyor. Ideally you you could test opencollective-postinstall for all node versions on osx/linux (travis) and windows (appveyor).

sheerun commented:
@xdamman Do you plan to update this PR?

xdamman commented:
Sorry no time to investigate Windows now so I just updated the package.json to remove the post install and replace it with a very simple "echo" in the console.

thinkxl commented:
It may be worth to get it working as it should in Windows. I will try to get a Windows machine to test it out.

sheerun commented:
Also, we can't use emoticons as some terminals break if one uses them: https://github.com/yarnpkg/yarn/issues/405

xdamman commented:
@sheerun ah good to know. I've updated the `opencollective-postinstall` package to not show emoji on Windows: https://github.com/OpenCollective/opencollective-postinstall/commit/b068519b21e50bcc97119769d8dc2c6b825dc147

I've also tested it with PowerShell on Mac and it works fine. But if someone could test on an actual Windows machine that would be great! Thank you for helping improving this package, it will benefit all collectives! :-)

xdamman commented:
So I made a bunch of changes to https://github.com/opencollective/opencollective-postinstall to make it work with Windows. It works but for some reason, I cannot make it work with node 0.10 that you are testing against. I'd need more time to investigate that (if anyone wants to chime in, help is welcome!)

In the meantime, I've just removed it and replaced it with a simple text without emoji.

sheerun commented on package.json:
```diff
@@ -91,10 +91,16 @@
     "coveralls": "coveralls",
-    "prepublish": "in-publish && echo 'You need to use \"grunt publish\" to publish bower' && false || not-in-publish"
+    "prepublish": "in-publish && echo 'You need to use \"grunt publish\" to publish bower' && false || not-in-publish",
+    "postinstall": "npm run -s donate",
```
bower can be installed with yarn, i'd rather not call it here

xdamman commented:
Ah. How could we do this?
Isn't npm coming with node anyway? Meaning that even if you install Bower with yarn, wouldn't this command still work?

The reason for doing it this way is to avoid having the command showing once and then execute which ends up showing that message twice. That's not super elegant.

sheerun commented:
something like `"postinstall": "node lib/postinstall.js"` would work everywhere

xdamman commented:
I pushed a change to fix this but for some reason appveyor didn't trigger.
Can you check?
